analytics/src/main/java/com/segment/analytics/AnalyticsActivityLifecycleCallbacks.java

```diff
@@ -50,14 +52,46 @@
   private AtomicInteger numberOfActivities;
   private AtomicBoolean firstLaunch;
 
+  private AtomicBoolean isChangingActivityConfigurations;
+  private Boolean useNewLifecycleMethods;
+
+  private static LifecycleOwner stubOwner =
```

Comment:
This is basically a stub which is only used when we dont use the new lifecycle flow

Reply:
required bcos some of the functions take in `NonNull` params, so this gets rid of those warnings